src/server/repository/assessmentCycle/countrySummary/getOneOrNone.ts

```diff
@@ -12,10 +13,11 @@ type Props = {
   assessment: Assessment
   countryIso: CountryIso
   cycle: Cycle
+  lang?: Lang
 }
 
 export const getOneOrNone = async (props: Props, client: BaseProtocol = DB): Promise<CountrySummary | null> => {
-  const { assessment, countryIso, cycle } = props
+  const { assessment, countryIso, cycle, lang = Lang.en } = props
```

Comment:
Logically to get one country the lang shouldn't be needed at all .

Would it be possible to add a parameter to the base query to exclude the lang sorting ?